common/src/main/java/tds/common/web/interceptors/EventLoggerInterceptor.java

```diff
@@ -26,7 +26,8 @@
 
     private final String app;
     private final ObjectMapper objectMapper;
-    private EventLogger eventLogger;
+    private final String EVENT_LOGGER_ATTRIBUTE = "event-logger";
```

Comment:
EventLoggerIntercetor has one instance per app. This meant that there was one eventLogger field per app.  For each http request, the eventLogger field was being replace by a new EventLogger instance.  The code worked until two calls http request happened at the same time.  The fix is to scope the new eventlogger to the request instead of the app.